lighthouse-core/index.js

```diff
@@ -39,7 +39,7 @@ module.exports = function(url, flags = {}, configJSON) {
 
     // kick off a lighthouse run
     return Runner.run(connection, {url, flags, config})
-      .then(lighthouseResults => {
+      .then((lighthouseResults = {}) => {
```

Comment:
what execution path leads to this being undefined?

Reply:
-G does. no lighthouse results because we saved artifacts and are resolving early

i could pass an object back there, but it seems slightly more awkward. open to ideas.